### PR TITLE
DEX-292 Allow any users created in POST /api/v1/users/ to login

### PR DESCRIPTION
**Pull request by karenc**
After creating users in frontend using `POST /api/v1/users/`, it was not
possible to login using the email and password.  The reason is because
in `POST /api/v1/auth/sessions` does not allow logins unless user is
`in_alpha`, `in_beta`, `is_staff` or `is_admin`.

### CODE REVIEW

File: tests/modules/auth/resources/test_general_access.py [hunk 1]
```diff
@@ -14,3 +16,28 @@ def test_unauthorized_access(http_method, http_path, flask_app_client):
     response = flask_app_client.open(method=http_method, path=http_path)
     print(response)
     assert response.status_code == 401
+
+
+def test_created_user_login(flask_app_client, admin_user, request):
+    from app.modules.users.models import User
```
Comment (brmscheiner) [inline tests/modules/auth/resources/test_general_access.py:22]: just curious - is it normal in python to put imports in the middle of a file? i've seen it a few times now and it always surprises me

Comment (karenc) [reply tests/modules/auth/resources/test_general_access.py:22]: No I wouldn't say it's normal but in houston there's some import loop (iirc) in `app/modules/users/models.py` and probably other places and it's not totally clear when it happens so in the end it's safer to import it in a function.

File: app/modules/users/resources.py [hunk 2]
```diff
@@ -83,6 +83,7 @@ def post(self, args):
             abort(code=HTTPStatus.BAD_REQUEST, message='Must provide a password')
 
         args['is_active'] = True
+        args['in_alpha'] = True
```
Comment (hwindsor) [inline app/modules/users/resources.py:86]: I understood that "alpha" meant "User has access to alpha release of product" so it seems wrong that this is being proposed as what any user has. 
I'd suggest asking @naknomum if this is the correct fix.

Comment (naknomum) [reply app/modules/users/resources.py:86]: yes, it seems to me _long-term_ we would want to: (a) modify login to allow in _some additional_ role of user(`active`?) (b) modify this part of the code to instead add _that_ role for admin-created users.

however, until we get some clarity on _that_, i dont see a huge problem with this as a sort of stop-gap measure.  alpha... or beta?   unless we wanna try to make the above decision before moving on with this.  maybe a good friday topic!

**Comment by brmscheiner** [comment]
works for me!

**Review by brmscheiner — APPROVED** [review]

File: app/modules/users/resources.py (hunk 2 above)
Comment (hwindsor) [reply app/modules/users/resources.py:86]: Surely the better fix is to change the auth code to allow any active user to log in. 
That's what active means isn't it?

**Comment by karenc** [comment]
We decided to remove this line https://github.com/WildMeOrg/houston/blob/a603fc4abdfa6c30dc0a9223afae3bed3b7c1057/app/modules/auth/resources.py#L62 in today's Houston Discord

**Review by hwindsor — APPROVED** [review]
LGTM

**Pull request merged**